### ISSUE TITLE
Readme.md

### ISSUE THREAD

**Issue by Marshud** [hakrichTech/HkmCode#1]
The readme needs to be updated and connect it to the project so it becomes usable

**Comment by hakrichTech** [comment]
Yes @Marshud i will work upon it but at first i would like to updates you guys about HkmCode installer in windows that am soon about to finish developing it so that i work in Readme file
